### PR TITLE
Adding instructions for uploading VoxelGPT to FiftyOne Teams

### PR DESCRIPTION
**Pull request by brimoor**

### CODE REVIEW

**Review by ritch — APPROVED** [review]

File: FIFTYONE_TEAMS.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+You can upload VoxelGPT manually or via the
+[Management SDK](https://docs.voxel51.com/teams/management_sdk.html).
+
+### Manual upload
+
+First clone and ZIP the repository:
+
+```shell
+git clone https://github.com/voxel51/voxelgpt
```
Comment (ritch) [inline FIFTYONE_TEAMS.md:16]: There's a link for this. Most other repos just link to that instead of having the user do it manually.

Comment (ritch) [reply FIFTYONE_TEAMS.md:16]: https://github.com/voxel51/voxelgpt/archive/refs/heads/main.zip

**Review by ritch — CHANGES_REQUESTED** [review]
Change zip instructions to download

File: FIFTYONE_TEAMS.md (hunk 1 above)
Comment (brimoor) [reply FIFTYONE_TEAMS.md:16]: Sweet 👍

File: FIFTYONE_TEAMS.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+```shell
+cat requirements.txt
+```
+
+If any of these Python packages are not available in your `teams-plugins`
+containers, you'll need to add them.
+
+You must also ensure that a valid OpenAI API key
+([create one](https://platform.openai.com/account/api-keys)) is available to
+the containers via the `OPENAI_API_KEY` environment variable.
```
Comment (findtopher) [inline FIFTYONE_TEAMS.md:62]: This seems reasonable... We could provide the Dockerfile we use and a snippet for Helm and compose if we wanted to make it even easier... 🤷

**Review by ritch — APPROVED** [review]

**Pull request merged**